### DISCUSSION TITLE
Disable escaping Brackets when sycning to markdown

### DISCUSSION THREAD

**Discussion by janschlatter** [windingwind/zotero-better-notes#301] [Q&A]
I'm trying to figure out how to make the sycn stop escape brackets [] with a \ in Markdown since these screw up linking in markdown format. I tried looking into the utils ts file but Im not quite finding the function responsible for handling this I presume regex function? Or is it handled by the remark library?

Anyone able to help guide me in the right direction?

**Comment by windingwind** [comment]
To keep the brackets during export/sync, I think an extra regex is a must. Currently, it’s controlled by the remark/rehype lib.

My concern is that this keeping brackets behavior can possibly spoil the syncing when users enter something like []() in the note. If they are kept, they will become a MD link and the original []() will be replaced after one syncing.

**Reply by janschlatter** [reply]
Thank you for your reply. I found issue #300 which is basically what I'm after as well, bummer it isnt as simple as I had hoped. Thanks for your work!

**Reply by windingwind** [reply]
Could you please share what is the case that you need to keep the bracket? If you want to paste a markdown link into your note, please try the Edit-Convert MD in Clipboard and then do the paste.

**Reply by BendeMorte** [reply]
I'm a proper noob and thus I might not properly understand how this is recommend to be done, however I face the issue when trying to have bi-directional links created in zotero through better-notes, function as backlinks in Obsidian.
The idea would be that other than opening the related note in Zotero (which is what the current links do when clicked on in Obsidian), the related note (which is also exported & synced with Obsidian) is opened inside Obsidian.
I'm using Mardown links [noteTitle](note Title.md) and have so far tried achieving this by configuring the [QuickBackLink] and [QuickInsert] templates to add the link in markdown format after the zotero format. 
The problem I'm facing is that when exported to markdown the '[', ']' are being escaped, braking the links in Obsidian.
A groundwork would be to have an Obsidian plugin unescaping these, but I'm not sure this would be compatibel with sync.

I know that the zotero export to markdown function doesn't escape '[' and was thus wondering if there exist way to achieve this behavior with the better-notes export function as well.

Thanks a lot for any help!
